common.blocks/checkbox-group/checkbox-group.bemhtml

```diff
@@ -29,6 +29,7 @@ block('checkbox-group')(
                     val : option.val,
                     text : option.text,
                     title : option.title,
+                    tabIndex : option.tabIndex || ctx.tabIndex,
```

Comment:
I'm not sure... There are cases when I need to fix tabIndex order inside a group. Can I redefine that in project level? That's not clear for me how to do that (because I'm not so familiar with bemhtml). If so I'm totally agree to left here `ctx.tabIndex` only.